tests/models/test_taskinstance.py

```diff
@@ -119,6 +121,17 @@ def setUp(self):
     def tearDown(self):
         self.clean_db()
 
+    def test_load_error_file_returns_None_for_closed_file(self):
+        error_fd = NamedTemporaryFile()
+        error_fd.close()
+        assert not load_error_file(error_fd)
```

Comment:
```suggestion
        assert load_error_file(error_fd) is None
```

Might be better to explicitly assert `None`?